src/engine/strat_engine/pool.rs

```diff
@@ -284,15 +284,19 @@ impl StratPool {
     /// Return an extend size for the physical space backing a pool
     /// TODO: returning the current size will double the space provisoned to
     /// back the pool.  We should determine if this is a reasonable value.
-    fn extend_size(&self, current_size: Sectors) -> Sectors {
+    fn extend_size(&self, current_size: DataBlocks) -> DataBlocks {
         current_size
     }
 
     /// Expand the physical space allocated to a pool by the value from extend_size()
-    /// Return tne number of Sectors added
-    fn extend_data(&mut self, dm: &DM, current_size: Sectors) -> EngineResult<Sectors> {
+    /// Return the number of DataBlocks added
+    fn extend_data(&mut self, dm: &DM, current_size: DataBlocks) -> EngineResult<DataBlocks> {
+        // The space backing the pool should be extended by a multiple of the
```

Comment:
You can drop this comment, it is implicit in the return type and explicit in the header comment.